docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -0,0 +1,207 @@
+# How to export a Kili project
+With Kili, once you have annotated enough assets, you can export the data programmatically to train a machine learning algorithm from it. There are several ways to do it:
+
+ * fetch the assets and/or the labels one by one using `.assets` or `.labels`, perform the data transformation yourself, then write them to one or several output files.
+ * export the whole project as a dataset using `.export_labels`, which will create an archive containing one or several files containing the labels in the format you need.
+
+
+## Preliminary steps
+ 1. Fetch the project ID from the Kili UI (in Settings / Admin):
+   ![Get Project ID from UI](../../assets/get_project_id.jpg)
+ 2. Ensure that your Kili API key as been set as an environment variable:
+ ```bash
+ export KILI_API_KEY = <YOUR_API_KEY>
+ ```
+ 3. Import packages and initialize the Kili client:
+```python
+from kili.client import Kili
+from pathlib import Path
+kili = Kili()
+```
+## Exporting assets and labels one by one
+To retrieve all assets of a project one by one, you can perform the following steps:
+
+### Exporting the latests labels per asset
+ 1. First fetch the assets:
+```python
+assets = kili.assets("<your_project_id>", fields=["externalId", "latestLabel.jsonResponse"])
+```
+
+ 2. Now if you print an asset, you will see that you can access its `latestLabel`:
+```python
+print(assets[0])
+{'latestLabel': {'jsonResponse': {'CLASSIFICATION_JOB': {'categories': [{'name': 'VEHICLE'}]}}}, 'externalId': '0'}
+```
+ 3. You can now get your label this way, and for example write the category name into a `0.txt` file:
+```python
+for asset in assets:
+    if asset["latestLabel"]: # covers the assets without label
+        class_ = asset["latestLabel"]["jsonResponse"]["CLASSIFICATION_JOB"]["categories"][0]["name"]
+        with (Path("/tmp") / (asset["externalId"] + ".txt")).open("w", encoding="utf-8") as f:
+            f.write(class_)
+```
+
+
+### Filtering specific labels per asset through the method filters
+You can specify label filters directly in the `.assets` and the `.labels` methods. The available filters are listed in the arguments
+of [`.assets`](https://python-sdk-docs.kili-technology.com/latest/sdk/asset/#kili.queries.asset.__init__.QueriesAsset.assets) and [`.labels`](https://python-sdk-docs.kili-technology.com/latest/sdk/label/#kili.queries.label.__init__.QueriesLabel.labels) methods.
+
+Once done, you can write the conversion code to obtain the data in the format you need.
+
+**Get all the assets, but each will only have the labels with a consensus mark above 0.7**
+```python
+assets = kili.assets("<your_project_id>", fields=["externalId", "labels.jsonResponse"], label_consensus_mark_gt=0.7)
```

Comment:
<your_project_id>  QA automation: we should probably replace items like this with variables.

Reply:
to be done when we will migrate towards the testable solution